src/components/sbb-toggle-option/sbb-toggle-option.tsx

```diff
@@ -61,6 +68,8 @@ export class SbbToggleOption implements ComponentInterface {
 
   @Element() private _element!: HTMLElement;
 
+  private _toggleOptionAttributeObserver = new MutationObserver(() => this._onToggleOptionChange());
```

Comment:
Why is in this case a "@Watch" on the checked attribute not possible? It would be good, if we document the reason of the approach somehow.

Reply:
I tried to use a `Watch('checked')` decorator but doing so would trigger a loop and break the page, with the MutationObserver I can check whether or not a `checked` attribute has simply been added to the DOM and call the `select()` method accordingly.

Reply:
Ok, I thought there must be something of this stuff ;) Just please comment in the code! thanks